.github/workflows/misc.yml

```diff
@@ -11,7 +11,7 @@ on:
       - v[0-9]+.x
 
 env:
-  NODE_VERSION: lts/*
+  NODE_VERSION: 14
```

Comment:
Probably would be good to include a comment so people don't update it to `lts/*` subsequently.

Reply:
Oh, wait, this is on the 14.x-staging branch. Yeah, that (probably) doesn't need a comment.